THIRD-PARTY-NOTICES

```diff
@@ -41,206 +41,18 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 License notice for Windows Azure Active Directory IdentityModel Extensions for .Net
 -----------------------------------------------------------------------------------
+Copyright (C) 2016-2020 SonarSource SA
```

Comment:
`Microsoft Corporation` rather than `SonarSource`?!

Reply:
Nope, take a look at "APPENDIX: How to apply the Apache License to your work."

Reply:
`Windows Azure Active Directory IdentityModel Extensions for .Net` isn't our work, it's Microsoft's. 
The appendix describes what Microsoft should write in their license file to apply the license to their work.

Section 4c says that we should retain Microsoft's copyright notice in our work when redistributing it.

Reply:
Updated, took their license from https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/dev/LICENSE.txt